content/python/concepts/sets/terms/intersection/intersection.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: '.intersection()'
+Description: 'Returns a new set with objects that exist inside two or more sets'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Functions'
+  - 'Methods'
+  - 'Sets'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `.intersection()` method returns a new set with objects that exist inside two or more [sets](https://www.codecademy.com/resources/docs/python/sets).
+
+## Syntax
+
+```pseudo
+set.intersection(set1, set2, .., setN)
+
+setA & setB & setC
+```
+
+The following syntaxes can be used:
```

Comment:
same as in the "difference.md" file; perhaps place this line before the pseudocode for the syntaxes.